news/2022-02-25-skinning-contest-tides-of-winter-results.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Skinning Contest: Tides of Winter - Results"
+date: 2022-02-25 12:00:00 +0000
+---
+
+As the Winter season gradually wraps up, the results of the chilliest skinning contest to date arrive at your doorstep warmer than ever!
```

Comment:
this makes way more sense in lowercase. if it was "Skinning Contest Winter Season" then yes, but not here

```suggestion
As the winter season gradually wraps up, the results of the chilliest skinning contest to date arrive at your doorstep warmer than ever!
```